Detections/SecurityAlert/Massdownload_USBFileCopy.yaml

```diff
@@ -0,0 +1,59 @@
+id: 6267ce44-1e9d-471b-9f1e-ae76a6b7aa84
+name: Mass download with new USB file copy activity
+description: |
+  'This query looks for any mass download by a single user with possible file copy activity to a new USB drive. Malicious insiders may perform such activities that may cause harm to the organization. This alert could also reveal unintentional insider that had no intention of malicious activity but their actions may impact an organizations security posture.'
```

Comment:
Can we please add a little more detail to the description that can probably help an analyst triage this better.
++ How does alert define mass download?
++ Is the download from M365/sharepoint or can it from general internet as well?
++ What security product surfaces this alert.
++ Any link to the alert description as reference.